dpnp/dpnp_iface_manipulation.py

```diff
@@ -583,12 +583,28 @@ def rollaxis(x1, axis, start=0):
     return call_origin(numpy.rollaxis, x1, axis, start)
 
 
-def squeeze(x1, axis=None):
+def squeeze(x, axis=None):
     """
     Remove single-dimensional entries from the shape of an array.
 
     For full documentation refer to :obj:`numpy.squeeze`.
 
+    Returns
```

Comment:
Below code generates different results, do you think it's an issue on dpctl?
```
dpnp.ones((1, 3, 1)).squeeze().flags.f_contiguous
Out: False

numpy.ones((1, 3, 1)).squeeze().flags.f_contiguous
Out: True
```

Reply:
I've filed an issue [#1180](https://github.com/IntelPython/dpctl/issues/1180) on dpctl.